src/services/ls-migration/addedSafes.ts

```diff
@@ -23,11 +26,33 @@ type OldAddedSafes = Record<
     address: string
     chainId: string
     ethBalance: string
-    owners: string[]
+    owners: Array<string | { name?: string; address: string }>
     threshold: number
   }
 >
 
+export const migrateAddedSafesOwners = (
+  owners: OldAddedSafes[string]['owners'],
+): AddedSafesState[string][string]['owners'] | undefined => {
+  const migratedOwners = owners
+    .map((value) => {
+      if (typeof value === 'string' && isChecksummedAddress(value)) {
+        return { value }
+      }
+
+      if (isObject(value) && typeof value.address === 'string' && isChecksummedAddress(value.address)) {
+        const owner: AddressEx = {
+          value: value.address,
+          ...(typeof value.name === 'string' && { name: value.name }),
```

Comment:
Do we use this name of the owners anywhere? 
I only saw places where we lookup the name in the address book.

Probably we should add the name to the address book if the address has no entry yet?

Reply:
Let's not complicate this, it's probably one out of a million users who will have this old format.

Reply:
> Do we use this name of the owners anywhere?

We seemingly don't, but our Safe creation does [add full `AddressEx` entries](https://github.com/safe-global/web-core/blob/dev/src/components/new-safe/create/logic/address-book.ts#L38) nonetheless.

I see no harm in keeping it. What do you think?

> Probably we should add the name to the address book if the address has no entry yet?

The address book is migrated separately. I don't think they should adjust the entries of one another.